mapreduce/src/test/java/org/pentaho/hadoop/mapreduce/PentahoMapRunnableTest.java

```diff
@@ -46,11 +46,17 @@
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.mock;
 
+import org.pentaho.di.core.logging.LogChannel;
+import org.pentaho.di.core.logging.LogChannelInterface;
+
 /**
  * @author Tatsiana_Kasiankova
  */
 @SuppressWarnings( { "unchecked", "rawtypes" } )
 public class PentahoMapRunnableTest {
+
+  private static LogChannelInterface log = new LogChannel( PentahoMapRunnableTest.class.getName() );
```

Comment:
Codacity is complaining about this field being unused; makes sense since the only line calling it is commented out.  Since this is test code anyway, I'd just ignore the whole thing and not bother updating the comment in it.

Reply:
Done